vtex/omnishipping
# CHK-462: Update styling of button in unavailability scenario

#### What is the purpose of this pull request?

Updates the styling of the "remove items from cart" button to be more distinct than the "pickup items" button.

#### What problem is this solving?

These two buttons were a lot similar to each other, which ended up confusing the user of the intent of each button.

#### How should this be manually tested?

[qastore workspace](https://lucas--qastore.myvtex.com/)
[vtexgame1 workspace](https://lucas--vtexgame1.myvtex.com/)

There are two main scenarios which you should test:

> Remember to delete you checkout.vtex.com cookie before entering any of these URLs!

1. [One item unavailable for delivery](https://lucas--qastore.myvtex.com/checkout/cart/add/?sku=454&qty=1&seller=1&sc=1&sku=455&qty=1&seller=1&sc=1&sku=564&qty=1&seller=1&sc=1), use the postal code 22250-040.
2. [One item unavailable for both delivery **and** pickup](https://lucas--qastore.myvtex.com/checkout/cart/add/?sku=454&qty=1&seller=1&sc=1&sku=564&qty=1&seller=1&sc=1&sku=452&qty=1&seller=1&sc=1), also use the postal code 22250-040.

#### Screenshots or example usage

![image](https://user-images.githubusercontent.com/10223856/100274509-4ca00c80-2f3d-11eb-8d21-96adefeacb35.png)
<p align="center"><i>Scenario 1</i></p>

![image](https://user-images.githubusercontent.com/10223856/100272755-6e4bc480-2f3a-11eb-8da2-c1d4977c77ad.png)
<p align="center"><i>Scenario 2</i></p>

#### Types of changes
